src/elements/content-preview/ContentPreview.js

```diff
@@ -194,6 +221,8 @@ class ContentPreview extends React.PureComponent<Props, State> {
 
     api: API;
 
+    boxAnnotations: Object;
```

Comment:
Is this instance property still needed?

Reply:
the webapp uses this to use v4 annotations, so it still needs to be here

Reply:
probably not, going to remove it